libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -973,6 +973,59 @@
+    
+    The output is 1 channel, with controls to adjust the output range.
+  -->
+  <nodedef name="ND_unifiednoise2d_float" node="unifiednoise2d" nodegroup="procedural2d">
+    <input name="texcoord" type="vector2" uifolder="Common" value="0, 0" />
+    <input name="freq" type="vector2" uiname="Frequency" uifolder="Common" value="1, 1" />
+    <input name="offset" type="vector2" uiname="Offset" uifolder="Common" value="0, 0" />
+    <input name="jitter" type="float" uiname="Jitter" uifolder="Common" uisoftmin="0.0" uisoftmax="1.0" value="1" />
+    <input name="outmin" type="float" uiname="Output Min" uifolder="Post Process" value="0" />
+    <input name="outmax" type="float" uiname="Output Max" uifolder="Post Process" value="1" />
+    <input name="outgamma" type="float" uiname="Output Gamma" uifolder="Post Process" uisoftmin="0.0" uisoftmax="3.0" value="1" />
+    <input name="clampoutput" type="boolean" uiname="Clamp Output" uifolder="Post Process" value="true" />
+    <input name="octaves" type="integer" uiname="Octaves" uifolder="Fractal" value="3" />
+    <input name="lacunarity" type="float" uiname="Lacunarity" uifolder="Fractal" value="2" />
+    <input name="diminish" type="float" uiname="Diminish" uifolder="Fractal" uisoftmin="0.0" uisoftmax="1.0" value="0.5" />
+    <input name="type" type="integer" uiname="Noise Type" uifolder="Common" uisoftmin="0" uisoftmax="3" value="0" enum="Noise,Cell,Worley,Fractal" enumvalues="0,1,2,3"/>
+    <output name="out" type="float" />
+  </nodedef>
+  
+  <!--
+    Node: <unifiednoise3d>
+    The 3d flavor of <unifiednoise2d>.
+  -->
+  <nodedef name="ND_unifiednoise3d_float" node="unifiednoise3d" nodegroup="procedural3d">
+    <input name="position" type="vector3" uifolder="Common" value="0, 0, 0" />
+    <input name="freq" type="vector3" uiname="Frequency" uifolder="Common" value="1, 1, 1" />
+    <input name="offset" type="vector3" uiname="Offset" uifolder="Common" value="0, 0, 0" />
+    <input name="jitter" type="float" uiname="Jitter" uifolder="Common" uisoftmin="0.0" uisoftmax="1.0" value="1" />
+    <input name="outmin" type="float" uiname="Output Min" uifolder="Post Process" value="0" />
+    <input name="outmax" type="float" uiname="Output Max" uifolder="Post Process" value="1" />
+    <input name="outgamma" type="float" uiname="Output Gamma" uifolder="Post Process" uisoftmin="0.0" uisoftmax="3.0" value="1" />
+    <input name="clampoutput" type="boolean" uiname="Clamp Output" uifolder="Post Process" value="true" />
+    <input name="octaves" type="integer" uiname="Octaves" uifolder="Fractal" value="3" />
+    <input name="lacunarity" type="float" uiname="Lacunarity" uifolder="Fractal" value="2" />
+    <input name="diminish" type="float" uiname="Diminish" uifolder="Fractal" uisoftmin="0.0" uisoftmax="1.0" value="0.5" />
+    <input name="type" type="integer" uiname="Noise Type" uifolder="Common" uisoftmin="0" uisoftmax="3" value="0" enum="Noise,Cell,Worley,Fractal" enumvalues="0,1,2,3"/>
```

Comment:
Would it be better to use "Perlin" vs "Noise" as the first option -- as it's all "noise" :).

Reply:
@kwokcb yes I agree! That was the original label, but I chose "noise" since the "noise2d" also omitted "perlin" from it's name was named. But yes I will change Noise to Perlin, thanks!